check.sh

```diff
@@ -0,0 +1,13 @@
+echo "Cloning opentelemetry-proto folder"
+svn checkout https://github.com/open-telemetry/opentelemetry-proto/trunk/opentelemetry
+mkdir proto
```

Comment:
We will probably want to add `proto/` to `.gitignore` so people don't inadvertently check it in.